### PR TITLE
Remove Facebook integration

### LINKED ISSUE #23: Facebook Integration broken
I don't know the details but I know it's not working.

### PR DESCRIPTION
**Pull request by luchaos**
Sending Facebook feed notifications hasn't worked as expected for the longest time.
Hides Facebook connect site awards.

Closes #23

### CODE REVIEW

File: lib/render/user.php [hunk 1]
```diff
@@ -220,11 +220,10 @@ function RenderSiteAwards($userAwards)
...
                 $linkdest = ''; //    TBD: referrals page?
-            } elseif ($awardType == 5) { //    Signed up for facebook!
-                $tooltip = "Awarded for associating their account with Facebook! Thanks for spreading the word!";
-
-                $imagepath = "/Images/_FBAssoc.png";
-                $linkdest = "/controlpanel.php#facebook";
+            // } elseif ($awardType == 5) { // Signed up for facebook!
+            //     $tooltip = "Awarded for associating their account with Facebook! Thanks for spreading the word!";
+            //     $imagepath = "/Images/_FBAssoc.png";
+            //     $linkdest = "/controlpanel.php#facebook";
```
Comment (Jamiras) [inline lib/render/user.php:226]: why comment out instead of remove?

File: public/reorderSiteAwards.php [hunk 2]
```diff
@@ -119,10 +119,10 @@
...
                     $imagepath = "/Images/00083.png";
                 }
-            } elseif ($awardType == 5) { //    Signed up for facebook!
-                $tooltip = "Awarded for associating their account with Facebook! Thanks for spreading the word!";
-                $awardTitle = "Facebook Association";
-                $imagepath = "/Images/_FBAssoc.png";
+                // } elseif ($awardType == 5) { //    Signed up for facebook!
+                //     $tooltip = "Awarded for associating their account with Facebook! Thanks for spreading the word!";
+                //     $awardTitle = "Facebook Association";
+                //     $imagepath = "/Images/_FBAssoc.png";
```
Comment (Jamiras) [inline public/reorderSiteAwards.php:125]: why comment out instead of remove?

**Review by Jamiras — APPROVED** [review]

File: lib/render/user.php (hunk 1 above)
Comment (luchaos) [reply lib/render/user.php]: I kept those for documentation purposes.`SiteAward` entries of that type still exist. 

Usually i'd remove dead code but this is the only reference to the magic number `5` - there are no DB comments on the respective column.

I went ahead and introduced the AwardType class which does the documentation job instead and removed those blocks.

File: public/reorderSiteAwards.php (hunk 2 above)
Comment (luchaos) [reply public/reorderSiteAwards.php:125]: See other comment

Comment (Jamiras) [reply public/reorderSiteAwards.php:125]: still present

File: lib/render/user.php (hunk 1 above)
Comment (Jamiras) [reply lib/render/user.php]: 👍

File: public/reorderSiteAwards.php (hunk 2 above)
Comment (luchaos) [reply public/reorderSiteAwards.php]: Oh my - copied code 🙈  Thanks!

**Review by Jamiras — APPROVED** [review]

**Pull request merged**